WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/FilterableCategoriesViewController.swift

```diff
@@ -137,14 +140,22 @@ extension FilterableCategoriesViewController: UITableViewDataSource {
         }
         cell.delegate = self
         cell.selectionStyle = UITableViewCell.SelectionStyle.none
-        cell.section = isLoading ? nil : visibleSections[indexPath.row]
-        cell.isGhostCell = isLoading
+
         if isLoading {
```

Comment:
The only real effective change here should be setting the `horizontalScrollOffset` - the rest was to make it clearer how cells should be configured when in "loading" mode versus not.

Reply:
Same observation about extracting the private method here